box/boxnavigator.py

```diff
@@ -220,7 +255,11 @@ def display(self, ax: plt.Axes, scale: float) -> None:
 
         # Check if the environment is of type TeleportingNavigator
         if isinstance(self, TeleportingNavigator):
-            self.draw_rectangle_ahead(ax, scale)  # Draw the rectangle
+            self.draw_current_past_rectangle(ax, scale)  # Draw the rectangle
+            ax.plot(self.anchor_1.x, self.anchor_1.y, "mx")
+            dxy = (self.anchor_1 - self.position).normalized() * scale
+            ax.plot(self.anchor_2.x, self.anchor_2.y, "mx")
+            dxy = (self.anchor_1 - self.position).normalized() * scale
```

Comment:
Probably worth documenting this or throwing it into a function.

Reply:
We don't need the dxy since as this was an old attempt at trying to make a cone to display the rotation that is valid. Similar to the red cone for the agent's current direction.